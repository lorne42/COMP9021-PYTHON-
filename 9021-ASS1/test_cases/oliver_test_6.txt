Here is your 
puzzle!
You 
have 
	met a 
group
of 4 islanders. Their names are Sirs Neil, Carol 
Zelda and Francine. Sir Carol says: "Sir Zelda is a 
Knight!" "Sir Carol is a Knight," says by Sir 
Francine.

	Sir Zelda speaks: "Sir Francine is a 
Knight." 	Sir Neil yells: "Sir Zelda and
 I are Knaves." Please solve the puzzle.