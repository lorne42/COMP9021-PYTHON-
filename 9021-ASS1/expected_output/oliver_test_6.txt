The Sirs are: Carol Francine Neil Zelda
There is a unique solution:
Sir Carol is a Knight.
Sir Francine is a Knight.
Sir Neil is a Knave.
Sir Zelda is a Knight.